config.lua

```diff
@@ -29,6 +29,10 @@ statusTimeout = 5000
 replaceKickOnLogin = true
 maxPacketsPerSecond = 25
 
+-- Store configuration
+storeImagesUrl = "http://static.tibia.com/images/store/"
```

Comment:
@brunominervino should explain somewhere that in the folder _/images/store/_ needs **have a folder named 64,** and the images needs to be **64x64** as well.